src/main/java/session/MenuSession.java

```diff
@@ -334,4 +335,19 @@ public boolean isOneQuestionPerScreen() {
     public void setOneQuestionPerScreen(boolean oneQuestionPerScreen) {
         this.oneQuestionPerScreen = oneQuestionPerScreen;
     }
+
+    public String[] getSessionStack() {
+        ArrayList<String> steps = new ArrayList<>();
+        SessionFrame frame = sessionWrapper.getFrame();
+        for (StackFrameStep step : frame.getSteps()) {
+            if (step.getType().equals(SessionFrame.STATE_COMMAND_ID)) {
+                steps.add("COMMAND: " + step.getId());
```

Comment:
to make this nicer for the frontend to parser it might make sense to return an array of objects, maybe something like
```
{
   stepType: 'command',
   value: 'm1-f0',
}
```